shared-libs/rules-engine/src/rules-emitter.js

```diff
@@ -145,6 +152,9 @@ const startSession = function() {
       session.match(err => {
         session.dispose();
         if (err) {
+          if (feedbackService) {
```

Comment:
This is probably a better place to make my null object request.... the conditional nesting here might easily be incorrectly "optimized" to be on a single line with an `&&` but of course that would create a bug. Removing this conditional steers clear of that entirely.